src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -654,8 +654,20 @@ function HistogramPlotWithControls({
             }}
           />
         </LabelledGroup>
-
-        <LabelledGroup label="X-axis">
+        {/* add vertical line in btw Y- and X- controls */}
```

Comment:
Maybe make the vertical separator a component?

Reply:
I think that perhaps it can be a technical debt or wait for viz refactoring that Dave mentioned before